src/Microsoft.AspNetCore/WebHost.cs

```diff
@@ -0,0 +1,179 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Reflection;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore
+{
+    /// <summary>
+    /// Provides convenience methods for creating instances of <see cref="IWebHost"/> and <see cref="IWebHostBuilder"/> with pre-configured defaults.
+    /// </summary>
+    public static class WebHost
```

Comment:
Oh then @davidfowl lied to me 😀
The other one is in an internal namespace so we can rename if need be. This is unlikely to ever cause naming conflicts in an app as ppl don't import our internal namespaces. Are there places in our code it causes an issue? I'd really like this type name for this class.